sources/packages/backend/apps/queue-consumers/test/helpers/testing-modules/testing-modules-helper.ts

```diff
@@ -0,0 +1,46 @@
+import { INestApplication } from "@nestjs/common";
+import { Test, TestingModule } from "@nestjs/testing";
+import { DataSource } from "typeorm";
+import { QueueConsumersModule } from "../../../src/queue-consumers.module";
+import { createZeebeModuleMock } from "@sims/test-utils/mocks/zeebe-client.mock";
+import { ZBClient } from "zeebe-node";
+import { SshService } from "@sims/integrations/services";
+import { createSSHServiceMock } from "@sims/test-utils";
+import * as Client from "ssh2-sftp-client";
+import { createMock } from "@golevelup/ts-jest";
+
+/**
+ * Result from a createTestingModule to support E2E tests creation.
+ */
+export class CreateTestingModuleResult {
+  nestApplication: INestApplication;
+  module: TestingModule;
+  dataSource: DataSource;
+  zbClient: ZBClient;
+  sshClientMock: Client;
+}
+
+/**
+ * Created the Queue Consumers root module needed to perform the E2E tests.
+ * @returns creation results with objects to support E2E tests.
+ */
+export async function createTestingAppModule(): Promise<CreateTestingModuleResult> {
+  const sshClientMock = createMock<Client>();
+  const module: TestingModule = await Test.createTestingModule({
+    imports: [QueueConsumersModule, createZeebeModuleMock()],
```

Comment:
Will the queueModule in QueueConsumersModule look for redis?

Reply:
I saw that the tests are using redis. Is there a way to tests the Processor class alone without involving redis? 
![image](https://user-images.githubusercontent.com/54600590/231570603-d1d1c077-6862-4085-a10c-eda20ede76bb.png)

Reply:
The current test files on this PR are not dependent on Redis.
I believe that we need to bring the `MockedQueueModule` from the API into lib and use it for both.

Reply:
As we discussed, Redis was there due to the BullModule connection created during module creation time. As discussed in the Team meeting, a workaround was created to allow the QueueModule to be entirely mocked and replaced.

Reply:
Thanks for doing the work to remove redis.